### ISSUE TITLE
jupyter console broken on latest conda-forge. Regression from jupyter_client-7.0.2 

### ISSUE THREAD

**Issue by slonik-az** [conda-forge/conda-forge.github.io#1511]
Regression against `jupyter_console-6.4.0` and ` jupyter_client-7.0.2 ` first reported in https://github.com/jupyter/jupyter_console/issues/251

**Comment by ocefpaf** [comment]
Pinging @conda-forge/jupyter_console and @conda-forge/jupyter_client teams.

**Comment by davidbrochart** [comment]
@slonik-az but it has nothing to do with conda-forge, does it?

**Comment by ocefpaf** [comment]
> @slonik-az but it has nothing to do with conda-forge, does it?

Even it it doesn't we need to know if we should move the packages to broken or not.

**Comment by davidbrochart** [comment]
I don't think so, I even opened [a PR in Jupyter Console](https://github.com/jupyter/jupyter_console/pull/244) ahead of jupyter_client's 7.0 release, but it was not merged.

**Comment by slonik-az** [comment]
> @slonik-az but it has nothing to do with conda-forge, does it?

The reason I cross-posted it here is that downgrading to `jupyter_client` 6.1.12 seems to resolve the issue with jupyter console. It is up to conda-forge to decide whether to roll back broken version or to label the package broken. In any case conda-force needs to be aware of what is going on.

**Comment by davidbrochart** [comment]
But which package/version do you consider broken?

**Comment by slonik-az** [comment]
> But which package/version do you consider broken?

IMHO, `jupyter console` is broken. Your patch has been available for review and merging since early July.
But who knows how long it will take to fix the jupyter console. In meanwhile, to preserve my workflow I downgraded `jupyter_client` to 6.1.12. Hope it won't break anything else. :-)

**Comment by davidbrochart** [comment]
Maybe Jupyter Console is just unmaintained, but I don't think we can mark it as broken. It's probably just a matter of pinning `jupyter_client <7.0` in its feedstock, until https://github.com/jupyter/jupyter_console/pull/244 is merged and released.

**Comment by slonik-az** [comment]
> Maybe Jupyter Console is just unmaintained,

I doubt that. `jupyter_console` is a dependency of`jupyter`. Also it is a very convenient way to connect  a terminal to a running kernel for debugging and what not.

**Comment by bollwyvl** [comment]
> `jupyter_console` is a dependency of `jupyter`

As a bit of history: the `jupyter` package is a historical artifact which preserves everything that was IPython prior to The Big Split, and as such brings in a _lot_ of stuff (e.g. qt), and without pins for much of anything, nor does it include a number of new, first-party jupyter components (e.g. `jupyterlab`).

**Comment by slonik-az** [comment]
Without `jupyter console` how else could I connect a terminal to a running kernel? IPython does not except `--existing` option any longer.
